jupyterhub/templates/ingress.yaml

```diff
@@ -9,11 +9,9 @@ metadata:
   name: {{ include "jupyterhub.ingress.fullname" . }}
   labels:
     {{- include "jupyterhub.labels" . | nindent 4 }}
-  {{- if .Values.ingress.annotations }}
+  {{- with .Values.ingress.annotations }}
   annotations:
-    {{- range $key, $value := .Values.ingress.annotations }}
-    {{ $key }}: {{ $value | quote }}
-    {{- end }}
+    {{- . | toYaml | trimSuffix "\n" | nindent 4 }}
```

Comment:
Actually, there could be a breaking changes here if someone has misused ingress.annotations having used a boolean value such as `true` instead of the string `"true"`.

If there is any such user, they would get an error from the k8s api-server about an invalid annotation.

Reply:
I think that's fine, if we're dropping helm 2 I think that should count as a breaking release anyway